### ISSUE TITLE
Is there any possible way to know the revolutions in motor mode?

### ISSUE THREAD

**Issue by ppfzamora** [ethanlipson/PyLX-16A#2]

**Comment by D-girl-11t** [comment]
No, but You can do the proper rotation in servo mode itself, by giving the angle and timing parameters. By doing so, it will automatically rotate at desired angular velocity.

Also, it would be great if there were angle constraints in motor mode. ( add this feature if possible )
